README.md

```diff
@@ -38,7 +38,7 @@
 | [chrome](https://www.ldoceonline.com/dictionary/chrome) [🔊](https://www.ldoceonline.com/media/english/ameProns/chrome.mp3) | ✅ [krəʊm] | ❌ [kram] |
 | [Daemon](https://www.ldoceonline.com/dictionary/Daemon) [🔊](https://www.ldoceonline.com/media/english/ameProns/daemon.mp3) | ✅ ['diːmən] | ❌ [[dæmən]] |
 | [digest](https://www.ldoceonline.com/dictionary/digest) [🔊](https://www.ldoceonline.com/media/english/ameProns/l3digest.mp3) | ✅ n. ['dɑɪdʒɛst] v. [dɑɪ'dʒɛst] | ❌ ['dɪgɛst] |
-| [Django](https://www.djangoproject.com/start/overview/) [🔊](https://www.google.com/search?q=django+pronunciation&oq=django+pronunciation&aqs=chrome.0.69i59j0i512l2j0i8i30l5j0i8i15i30j0i5i30.6418j0j7&sourceid=chrome&ie=UTF-8) | ✅ [ˈdʒæŋɡoʊ], JANG-goh| ❌ [diˈdʒæŋɡoʊ] |
+| [Django](https://www.djangoproject.com/start/overview/) [🔊](https://www.google.com/search?q=django+pronunciation&sxsrf=ALiCzsZzksRFniiyFpwrCBPmruIjBSVmqg%3A1654444456635&ei=qNGcYrm5Joyi0gTd-qqoAQ&oq=Django+pron&gs_lcp=Cgdnd3Mtd2l6EAEYADIFCAAQywEyBQgAEIAEMgUIABDLATIFCAAQywEyBQgAEMsBMgUIABDLATIFCAAQywEyBQgAEMsBMgQIABAeMgYIABAeEA86BwgAEEcQsAM6BAgAEEM6CgguELEDEIMBEENKBAhBGABKBAhGGABQ_gJYjDJguUBoAXABeACAAVOIAZYCkgEBNZgBAKABAcgBCsABAQ&sclient=gws-wiz) | ✅ [ˈdʒæŋɡoʊ], JANG-goh| ❌ [diˈdʒæŋɡoʊ] |
```

Comment:
If there any reason why the need to add these params?

Reply:
No idea, I think this just a redundant query. I will push a new commit.

Reply:
Updated thx! 👌